proposals/records.md

```diff
@@ -40,28 +40,37 @@ The synthesized members are as follows:
 
 Record types produce synthesized implementations for the following methods, where `T` is the
 containing type:
+```C#
+public override int GetHashCode();
```

Comment:
>GetHashCode [](start = 20, length = 11)

Are these methods intended to override the similar methods from `object`? If so, you need to say that, as there could be an intervening method hiding the one in `object`.